modpack_format/modpack_example.modip.json

```diff
@@ -0,0 +1,35 @@
+{
+  "formatType": "modipModpack",
+  "formatVersion": "1.0.0",
+  "id": "modpack-example",
+  "name": "Modpack Example",
+  "summary": "Summary of the pack",
+  "description": "Detailed description of the modpack. <b>HTML?</b>",
+  
+  "updates": "https://api.example.com/pack/modpack-example.modip.json",
```

Comment:
What does the launcher do with this?

Reply:
I haven't written a launcher implementation guide yet. My basic thoughts are that when a user clicks a "check for updates" button or similar, the launcher will request this.

Reply:
and do what with it?

Reply:
Look for the most recent version and compare it's release date timestamp with the currently installed version. Which reminds me, I need to add `releaseDate` field.

Reply:
And if it's different... do what? And why do I have to ask these questions instead of getting the whole thing in one answer?

Reply:
Because I thought it was obvious but I guess not. If it's newer than the current timestamp then it should give the user the option to update. The actual process of updating and figuring out conflicts is the responsibility of the launcher.

Reply:
You still have not said what the URL points to. A release date? After the launcher downloads the release date from the URL and compares it to the release date in the JSON file, how does it know how to update the pack?

Reply:
It will point to a JSON file that lists all the versions of a modpack as well as their release timestamp, probably similar to #9. I have not written that though, which is why this is currently a draft.

Reply:
Won't peterix say no, this is too complicated?

Reply:
I don't think it's too likely. Updating modpacks is a convenience feature that users want, and it shouldn't be too complicated to implement. The actual format that will be returned from the url will be fairly simple, I just need to get around to writing it.